notebooks/239-freevc/239-freevc.ipynb

```diff
@@ -0,0 +1,733 @@
+{
```

Comment:
<p>can you please provide more info about model? Which parts it consist? In which paper described?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>